### DISCUSSION TITLE
just a WIP mockup I did a while ago for the UI

### DISCUSSION THREAD

**Discussion by brigazvi** [MFEK/glif#293] [Ideas]
didn't find the time to finish it. the right panel should be for layers and there should be an adaptive top bar for tool options.

![Desktop - 1](https://user-images.githubusercontent.com/37640970/170936425-f91e15be-e297-4e21-832c-551419f8e867.png)

**Comment by ctrlcctrlv** [comment]
That looks really cool. :)
